[2025-07-07T07:19:28.449+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-07T07:19:28.468+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: minio_ga4_new_file_detector.detect_new_ga4_data manual__2025-07-07T06:53:38.076557+00:00 [queued]>
[2025-07-07T07:19:28.472+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: minio_ga4_new_file_detector.detect_new_ga4_data manual__2025-07-07T06:53:38.076557+00:00 [queued]>
[2025-07-07T07:19:28.472+0000] {taskinstance.py:2303} INFO - Starting attempt 10 of 11
[2025-07-07T07:19:28.477+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): detect_new_ga4_data> on 2025-07-07 06:53:38.076557+00:00
[2025-07-07T07:19:28.481+0000] {standard_task_runner.py:63} INFO - Started process 3941 to run task
[2025-07-07T07:19:28.484+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'minio_ga4_new_file_detector', 'detect_new_ga4_data', 'manual__2025-07-07T06:53:38.076557+00:00', '--job-id', '153', '--raw', '--subdir', 'DAGS_FOLDER/dag_automatic.py', '--cfg-path', '/tmp/tmp3temu75m']
[2025-07-07T07:19:28.485+0000] {standard_task_runner.py:91} INFO - Job 153: Subtask detect_new_ga4_data
[2025-07-07T07:19:28.498+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-07T07:19:28.522+0000] {task_command.py:426} INFO - Running <TaskInstance: minio_ga4_new_file_detector.detect_new_ga4_data manual__2025-07-07T06:53:38.076557+00:00 [running]> on host 34b334fe6b50
[2025-07-07T07:19:28.572+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='minio_ga4_new_file_detector' AIRFLOW_CTX_TASK_ID='detect_new_ga4_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T06:53:38.076557+00:00' AIRFLOW_CTX_TRY_NUMBER='10' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-07T06:53:38.076557+00:00'
[2025-07-07T07:19:28.572+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-07T07:19:28.631+0000] {logging_mixin.py:188} INFO - B·∫Øt ƒë·∫ßu ki·ªÉm tra c√°c file m·ªõi h∆°n timestamp: 2025-07-07T07:17:37.407000+00:00
[2025-07-07T07:19:28.643+0000] {logging_mixin.py:188} INFO - üí§ Kh√¥ng t√¨m th·∫•y d·ªØ li·ªáu GA4 m·ªõi n√†o.
[2025-07-07T07:19:28.644+0000] {python.py:237} INFO - Done. Returned value was: False
[2025-07-07T07:19:28.644+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-07T07:19:28.663+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=minio_ga4_new_file_detector, task_id=detect_new_ga4_data, execution_date=20250707T065338, start_date=20250707T071928, end_date=20250707T071928
[2025-07-07T07:19:28.695+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-07T07:19:28.716+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-07T07:19:28.717+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
